test/e2e/antreapolicy_test.go

```diff
@@ -49,11 +49,12 @@ var (
 	podsByNamespace                             map[string][]Pod
 	k8sUtils                                    *KubernetesUtils
 	allTestList                                 []*TestCase
-	pods                                        []string
-	namespaces                                  map[string]string
+	podsPerNamespace                            []string
+	namespaces                                  map[string]TestNamespaceMeta
 	podIPs                                      map[string][]string
 	p80, p81, p8080, p8081, p8082, p8085, p6443 int32
 	nodes                                       map[string]string
+	selfNamespace                               *crdv1beta1.PeerNamespaces
```

Comment:
This is a "invariant" variable defined for reuse, right? I think it's more clear to initialize its value here, instead of in a function. So do other cases like "p80, p81, p8080, p8081, p8082, p8085, p6443".